File: src/Entity/Feedback.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Feedback
 *
 * @ORM\Table(name="feedback")
 * @ORM\Entity
 */
class Feedback
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="sujet", type="string", length=100, nullable=false)
     */
    private $sujet;

    /**
     * @var string
     *
     * @ORM\Column(name="description", type="string", length=255, nullable=false)
     */
    private $description;

    /**
     * @var string
     *
     * @ORM\Column(name="type", type="string", length=255, nullable=false)
     */
    private $type;

    /**
     * @var int
     *
     * @ORM\Column(name="nbr_etoiles", type="integer", nullable=false)
     */
    private $nbrEtoiles;

    /**
     * @var int|null
     *
     * @ORM\Column(name="OtherID", type="integer", nullable=true, options={"default"="NULL"})
     */
    private $otherid = NULL;

    /**
     * Feedback constructor.
     * @param int $id
     * @param string $sujet
     * @param string $description
     * @param string $type
     * @param int $nbrEtoiles

     */
    public function __construct()
    {

    }

    /**
     * @return int
     */
    public function getId(): ?int
    {
        return $this->id;
    }

    /**
     * @param int $id
     */
    public function setId(int $id): void
    {
        $this->id = $id;
    }

    /**
     * @return string
     */
    public function getSujet(): ?string
    {
        return $this->sujet;
    }

    /**
     * @param string $sujet
     */
    public function setSujet(string $sujet): void
    {
        $this->sujet = $sujet;
    }

    /**
     * @return string
     */
    public function getDescription(): ?string
    {
        return $this->description;
    }

    /**
     * @param string $description
     */
    public function setDescription(string $description): void
    {
        $this->description = $description;
    }

    /**
     * @return string
     */
    public function getType(): ?string
    {
        return $this->type;
    }

    /**
     * @param string $type
     */
    public function setType(string $type): void
    {
        $this->type = $type;
    }

    /**
     * @return int
     */
    public function getNbrEtoiles(): ?int
    {
        return $this->nbrEtoiles;
    }

    /**
     * @param int $nbrEtoiles
     */
    public function setNbrEtoiles(int $nbrEtoiles): void
    {
        $this->nbrEtoiles = $nbrEtoiles;
    }

    /**
     * @return int|null
     */
    public function getOtherid(): ?int
    {
        return $this->otherid;
    }

    /**
     * @param int|null $otherid
     */
    public function setOtherid(?int $otherid): void
    {
        $this->otherid = $otherid;
    }


}

File: SlowlifeMobile (2)/SlowlifeMobile/test/src/com/mycompany/entities/Feedback.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.mycompany.entities;

/**
 *
 * @author Ahmed Ezzine
 */
public class Feedback {
    private int id;
    private String sujet;
    private int rate;
    private String Description;
    private String type;
    private int otherid;
    private int statRate;

    public Feedback() {
    }

    public Feedback(String sujet, int rate, String Description, String type) {
        this.sujet = sujet;
        this.rate = rate;
        this.Description = Description;
        this.type = type;
    }

    public Feedback(String sujet, int rate, String Description, String type, int otherid) {
        this.sujet = sujet;
        this.rate = rate;
        this.Description = Description;
        this.type = type;
        this.otherid = otherid;
    }

    public Feedback(int id, String sujet, int rate, String Description, String type, int otherid) {
        this.id = id;
        this.sujet = sujet;
        this.rate = rate;
        this.Description = Description;
        this.type = type;
        this.otherid = otherid;
    }

    public Feedback(int id) {
        this.id = id;
    }

    public Feedback(String toString, String toString0, String toString1, String valueOf) {
        throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getSujet() {
        return sujet;
    }

    public void setSujet(String sujet) {
        this.sujet = sujet;
    }

    public int getRate() {
        return rate;
    }

    public void setRate(int rate) {
        this.rate = rate;
    }

    public String getDescription() {
        return Description;
    }

    public void setDescription(String Description) {
        this.Description = Description;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public int getOtherid() {
        return otherid;
    }

    public void setOtherid(int otherid) {
        this.otherid = otherid;
    }

    @Override
    public String toString() {
        return "Feedback{" + "id=" + id + ", sujet=" + sujet + ", rate=" + rate + ", Description=" + Description + ", type=" + type + ", otherid=" + otherid + '}';
    }
    
}
